app/src/main/java/com/kickstarter/services/KSApolloClientV2.kt

```diff
@@ -1944,6 +1946,27 @@ class KSApolloClientV2(val service: ApolloClient, val gson: Gson) : ApolloClient
         } ?: emptyList()
     }
 
+    override suspend fun getLocations(useDefault: Boolean, term: String?): Result<List<Location>> = executeForResult {
```

Comment:
📥  Still WIP!! hardcoded for now, the running app does not call any of this methods, just the compose preview